homeassistant/components/hassio/services.yaml

```
@@ -1,37 +1,84 @@
 addon_install:
-  description: Install a HassIO docker addon.
+  description: Install a HassIO docker add-on.
```

Sorry! Missed that the first round, `HassIO` -> `Hass.io`.
This applies to the rest as well.

```suggestion
  description: Install a Hass.io docker add-on.
```

---

I miss this one too.